packages/components/src/components/TaskTree/TaskTree.js

```diff
@@ -35,8 +35,11 @@ class TaskTree extends Component {
             return null;
           }
           const { id, reason, steps, succeeded, pipelineTaskName } = taskRun;
+          const erroredTask = taskRuns.find(task => task.succeeded === 'False');
           const expanded =
-            selectedTaskId === id || (!selectedTaskId && index === 0);
+            succeeded === 'False' ||
```

Comment:
this results in all failed tasks being expanded instead of just selecting and expanding the first failed one...
![image](https://user-images.githubusercontent.com/2829095/72067334-5b0c1800-32da-11ea-9f5f-0cfdda3a90b5.png)

Reply:
🧐🧐🧐 good catch ill work on it.